app/models/event.rb

```diff
@@ -72,7 +72,7 @@ def self.events_to_json
   end
 
   def self.events_to_calendar(format = nil)
-    events = Event.all
+    events = Event.where(is_provisional_date: FALSE)
```

Comment:
Could you please lowercase true/false boolean? So we keep the code according to a standard

Reply:
Sorry! I will fix that @Levia !